meson.build

```diff
@@ -195,7 +197,16 @@ config_h = configure_file(
   configuration: conf_data
 )
 
+sys_magic = cc.find_library('magic', required: false)
+use_lib_magic = false
+
+if sys_magic.found() and get_option('use_sys_magic')
+  use_lib_magic = true
+endif
+
 userconf = configuration_data()
+userconf.set('HAVE_LIB_MAGIC', sys_magic.found().to_int())
```

Comment:
`r_userconf.h.acr` doesn't contin this macros

Reply:
i see it https://github.com/radare/radare2/blob/master/libr/include/r_userconf.h.acr#L79

Reply:
Ok my mistake